src/components/Error/NotFound.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import './Error.css'
```

Comment:
Would it be better to dynamically pass in a "message" as props to this error component, and have one of those messages be 404: page not found?